.changeset/seven-tomatoes-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-backend': minor
+---
+
+fixes api auth bug in techdocs backend
```

Comment:
Probably good to mention this bug only affected the `DefaultTechDocsCollator` and not the backend itself.